lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -325,6 +329,7 @@ def create_tar(cv, cvv, repositories, json)
         Dir.mkdir("#{export_dir}/#{export_prefix}")
 
         if repositories
```

Comment:
I think you still need to check if `repositories&.any?` here, as an empty repositories array (`[]`) will still be exported.

Reply:
I think at this point empty repositories array will not be present because we are checking for empty repositories in the previous function. 

```
if repositories&.any? or cv['composite']
     create_tar(cv, cvv, repositories, json)
else
     puts "Ensure the content view version '#{cvv['name']}' has at least one repository."
end
```

Reply:
Could repositories still be `[]` though if it's a composite CV?

Reply:
@akofink I do not think that repository can be `[]` in case of composite CV. 
Repository return `[]` only if any content view is published without adding any repository in it. 
In can of composite CV, I have observed that repository variable itself is not set. But anyways its good change it to `repositories&.any?`. I will make the changes and send the commit again.